alonzo/formal-spec/utxo.tex

```diff
@@ -54,16 +51,14 @@ \section{UTxO}
   \begin{enumerate}[label=({\roman*})]
     \item the fee amount that the transaction states it is paying suffices to cover
     the minimum fee that the transaction is obligated to pay; and if the transaction uses non-native scripts, that
-    \item the fee-marked inputs do not belong to non-native script addresses (that is, they
-    belong to either a native script address, or a key hash one);
-    \item all the fee-marked inputs contain strictly Ada and no other kinds of token; and
-    \item the fee-marked inputs are sufficient to cover the fee amount that is stated
-    in the transaction.
+    \item the collateral inputs are strictly in key-hash addresses;
+    \item all the collateral inputs contain strictly Ada and no other kinds of token; and
+    \item the collateral inputs are sufficient to cover the percentage of the
+    minimum fee amount specified by the protocol parameter $\fun{collateralPercent}$
```

Comment:
minumum fee amount or the fee amount given in the transaction? I had assumed the latter.

Reply:
hmm... i guess probably the fee given by transaction is better